src/Sarif.Multitool/PageOptions.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    /// <summary>
+    ///  Options for the 'Page' command, which quickly writes a subset of a SARIF file
+    ///  for easier consumption of huge files.
+    /// </summary>
+    /// <remarks>
+    ///  Excluded Options
+    ///  ================
+    ///    pretty-print: We copy slices of the input file, so we can't change formatting.
+    ///    inline: We build a map of the input, so we don't want to write inline and immediately invalidate it.
+    /// </remarks>
+    [Verb("page", HelpText = "Extract a subset of results from a source SARIF file.")]
+    internal class PageOptions
+    {
+        [Value(0,
+            MetaName = "<inputFile>",
+            HelpText = "A path to a SARIF file to process.",
+            Required = true)]
+        public string InputFilePath { get; set; }
+
+        [Option('r',
+            "run-index",
+            HelpText = "0-based index of the run to copy.",
+            Required = false,
+            Default = 0)]
+        public int RunIndex { get; set; }
+
+        [Option('i',
+            "index",
+            HelpText = "0-based index of first result to include.",
+            Required = true)]
+        public int Index { get; set; }
+
+        [Option('c',
+            "count",
+            HelpText = "Number of results from index to include.",
+            Required = true)]
+        public int Count { get; set; }
+
+        // Force defaults to true, so users creating many large pages to the same output file name
+        // won't actually be getting the same page repeatedly without knowing it.
+        [Option(
+            'f',
+            "force",
+            Default = true,
+            HelpText = "Force overwrite of output file if it exists.")]
+        public bool Force { get; set; } = true;
+
+        [Option(
+            'o',
+            "output",
+            HelpText = "A file path of the SARIF subset output file.",
+            Required = true)]
+        public string OutputFilePath { get; set; }
+
+        [Option('a',
```

Comment:
>a [](start = 17, length = 1)

why 'a'? maybe 'm' would be more intuitive?

Reply:
Good suggestion - the result of an earlier rename.
I've removed the short name for this argument, as I think it'll be rarely used and isn't especially intuitive.